### PR TITLE
Integrate data at 0x0098DA64

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/object_list.c [hunk 1]
```diff
@@ -130,14 +133,14 @@ void object_create_identifier_name(char* string_buffer, const rct_object_entry*
 
 	for (uint8 i = 0; i < 4; ++i){
```
Comment (duncanspumpkin) [inline src/object_list.c]: This function should have just been replaced with `sprintf(string_buffer, "%.8s/%4X%4X", object->name, object->flags, object->checksum);`

**Comment by marijnvdwerf** [comment]
Tried that. Caused the hash to be in reverse order.
Op zo 18 sep. 2016 om 22:06 schreef Duncan <notifications@github.com>

**Comment by duncanspumpkin** [comment]
Its just used for the error message it doesnt matter the order.

File: src/object_list.c (hunk 1 above)
Comment (IntelOrca) [inline src/object_list.c]: Done!

**Pull request merged**